cub/cub/thread/thread_operators.cuh

```diff
@@ -38,6 +38,8 @@
 #pragma once
 
 #include <cub/config.cuh>
+#include <cub/detail/type_traits.cuh>   // always_false_v
+#include "cuda/std/__cccl/attributes.h" // _CCCL_NODISCARD
```

Comment:
Ditto already in `config`. Generally all macros that go `_CCCL` are available globally
```suggestion
```